backend-rust/src/indexer.rs

```diff
@@ -676,6 +692,7 @@ struct BlockData {
     tokenomics_info:      RewardsOverview,
     total_staked_capital: Amount,
     special_events:       Vec<SpecialTransactionOutcome>,
+    /// Certificates included in the block.
```

Comment:
Thisis kind of given by the name. :)

Reply:
Well, it could have been the certificates related to this block and not the ones are included in the block.